src/partials/assertion_required.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+  {{> back_button}}
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo"/>
+  </div>
+
+  <div style="display: none" id="unsupportedDeviceId" class="icon-feedback icon-feedback--timeout icon-feedback--error">
+    <span class="icon-feedback__icon pingicon-ghost-error"></span>
+    <p class="icon-feedback__label">
+      <p class="icon-feedback--error">Unsupported authenticator</p>
+    </p>
+  </div>
+
+  <div style="display: none" id="consentRefusedId" class="icon-feedback icon-feedback--timeout icon-feedback--error">
+    <span class="icon-feedback__icon pingicon-ghost-error"></span>
+    <p class="icon-feedback__label">
+      <p class="icon-feedback--error">Unsupported authenticator or user refused consent</p>
+    </p>
+  </div>
+
+  <form class="form" id="AuthnWidgetForm" data-actionId="checkAssertion">
+    {{> errors}}
+    {{> spinner_assertion_required}}
```

Comment:
If this spinner is just for this template I think we should just include it here. 

I think this is needed because we want the spinner to display immediately? We might just need a common component for that as other templates have needed the same thing

Reply:
Yes that's the behaviour. I have include the spinner here for now and remove other file.